elasticsearch/probe/readiness.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright 2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO: we need to re-use code that is currently found in ./run.sh
+ES_REST_BASEURL=https://localhost:9200
+secret_dir=/etc/elasticsearch/secret
+max_time=300
```

Comment:
I think the timeout in this context should be `1`.  Perhaps something like this:
```
max_time=${max_time:-1}
```
and let k8s just retry periodically until it hits its max timeout or max failures.